docs/get_started.rst

```diff
@@ -1,17 +1,341 @@
+
+Next, run the following command and provide your GitHub username and the access token as the password::
+
+    $ cci service connect github
+
+You can verify the GitHub service is connected by running ``cci service list``:
+
+.. image:: images/service-list.png
+
+Once you've configured the ``github`` service it will be available to **all** CumulusCI projects.
+Services are stored in the global CumulusCI keychain by default.
+
+
 
-Work on an existing CumulusCI project
+Work on an Existing CumulusCI Project
 -------------------------------------
+Before working on an existing CumulusCI project you need to:
+
+* `Install CumulusCI`_
+* `Install git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+
+
+
+Cloning a GitHub Repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The first step in working on an existing project is cloning a projects repository.
+Cloning gives you a local working copy of the project on your computer.
+
+To clone a GitGub repository:
+
+#.  Navigate to the repository on GitHub
+#.  Click the green 'Code' button
+#.  Ensure 'HTTPS' is selected #TODO: cover ssh or gh cli?
+#.  Click the clipboard button to copy the https repository url to your clipboard.
+#.  In a new terminal window exectue the following command:
+
+.. code-block:: console
+
+    $ git clone <repository_url> <project_name>
+
+Replace ``<repository_url>`` with the url copied to your clipboard.
+Replace ``<project_name>`` with the name of the project.
+
+You can now change directories into the freshly cloned project and begin executing ``cci`` commands.
+For example, ``cci project info`` can be run to display information about the project:
+
+.. code-block:: console
+
+    $ cd cumulusci-test
+
+    $ cci project info
+    name: CumulusCI Test
+    package:
+        name: CumulusCI Test
+        name_managed: None
+        namespace: ccitest
+        install_class: None
+        uninstall_class: None
+        api_version: 33.0
+    git:
+        default_branch: main
+        prefix_feature: feature/
+        prefix_beta: beta/
+        prefix_release: release/
+        release_notes:
+            parsers:
+                1:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Critical Changes
+                2:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Changes
+                3:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubIssuesParser
+                    title: Issues Closed
+                4:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: New Metadata
+                5:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Deleted Metadata
+        repo_url: https://github.com/SFDO-Tooling/CumulusCI-Test
+    test:
+        name_match: %_TEST%
+
+
```

Comment:
we should have a link here to direct them to the next section of the docs, since the rest of this page is for different starting scenarios.